cryptoconditions/types/ed25519.py

```diff
@@ -57,7 +57,7 @@ def sign(self, message, private_key):
             private_key (string) Ed25519 private key
         """
         sk = private_key
-        vk = VerifyingKey(base58.b58encode(sk.get_verifying_key().to_bytes()))
+        vk = VerifyingKey(base58.b58encode(sk.get_verifying_key().encode(encoding='bytes')))
```

Comment:
`get_verifying_key` was already part of the `Ed25519SigningKey` api. Didn't want to change it since its also used in the rest of the cc code.